juniper/src/tests/fixtures/tracing/schema.rs

```diff
@@ -0,0 +1,551 @@
+    fn with_arg(
+        &self,
+        id: i32,
+        skipped: i32,
+        #[graphql(default = 0)] default: i32,
+        #[graphql(default = 1)] overwritten: i32,
+    ) -> i32 {
+        id + skipped + default + overwritten
+    }
+
+    /// Async interface field with various arguments.
+    #[tracing(skip(skipped))]
+    async fn async_with_arg(
+        &self,
+        id: i32,
+        skipped: i32,
+        #[graphql(default = 0)] default: i32,
+        #[graphql(default = 1)] overwritten: i32,
+    ) -> i32 {
+        id + skipped + default + overwritten
+    }
+
+    /// Field with overwritten `target` of span.
+    #[tracing(target = "my_target")]
+    fn target(&self) -> i32 {
+        1
+    }
+
+    /// Field with overwritten `level` of span.
+    #[tracing(level = "warn")]
+    fn level(&self) -> i32 {
+        2
+    }
+}
+
+/// GraphQL object marked with `trace = "skip-sync"`
+pub struct TraceSync;
+
+#[graphql_object(
+    trace = "sync",
+    impl = [InterfacedSimpleValue, InterfacedSyncValue],
+)]
+impl TraceSync {
+    pub fn sync_fn(&self) -> i32 {
+        1
+    }
+
+    pub async fn async_fn(&self) -> i32 {
+        2
+    }
+}
+
+/// Derived GraphQL object marked with `trace = "sync"`.
+#[derive(Default, GraphQLObject)]
+#[graphql(trace = "sync")]
+pub struct SyncDerived {
+    /// Simple field
+    sync: i32,
+}
+
+/// GraphQL object marked with `trace = "async"`.
+pub struct TraceAsync;
+
+#[graphql_object(
+    trace = "async",
+    impl = [InterfacedAsyncValue],
+)]
+impl TraceAsync {
+    pub fn sync_fn(&self) -> i32 {
+        1
+    }
+
+    pub async fn async_fn(&self) -> i32 {
+        2
+    }
+}
+
+/// Derived GraphQL object
+#[derive(Default, GraphQLObject)]
+#[graphql(trace = "async")]
+pub struct AsyncDerived {
+    /// Simple field
+    sync: i32,
+}
+
+/// GraphQL object marked with `trace = "skip-all"`.
+pub struct SkipAll;
+
+#[graphql_object(
+    trace = "skip-all",
+    impl = [InterfacedSkipAllValue],
+)]
+impl SkipAll {
+    pub fn sync_fn(&self) -> i32 {
+        1
+    }
+
+    pub async fn async_fn(&self) -> i32 {
+        2
+    }
+}
+
+/// Derived GraphQL object marked with `trace = "skip-all"`
+#[derive(Default, GraphQLObject)]
+#[graphql(trace = "skip-all")]
+pub struct SkipAllDerived {
+    /// Simple field
+    sync: i32,
+}
+
+/// GraphQL object marked with `trace = "complex"`
+pub struct Complex;
+
+#[graphql_object(
+    trace = "complex",
+    impl = [InterfacedComplexValue],
+)]
+impl Complex {
+    #[tracing(complex)]
+    pub fn sync_fn(&self) -> i32 {
+        1
+    }
+
+    #[tracing(complex)]
+    pub async fn async_fn(&self) -> i32 {
+        2
+    }
+
+    fn simple_field(&self) -> i32 {
+        3
+    }
+
+    #[tracing(complex, no_trace)]
+    fn no_trace_complex(&self) -> i32 {
+        4
+    }
+}
+
+/// Derived GraphQL object marked with `trace = "complex"`.
+#[derive(GraphQLObject)]
+#[graphql(trace = "complex")]
+pub struct DerivedComplex {
```

Comment:
@ArsileLuci I think we can use here `#[tracing(complex)]` too.

Also, let's check whether we can do that for `#[graphql_object]` macro, by looking up this attribute on a definition, parsing it and removing it. If not, it would be better to keep the similar to other cases format:
```rust
#[graphql_object(tracing(complex))]
```

This should also simplify the parsing, as we would have similar patterns everywhere.